packages/py-moose-lib/setup.py

```diff
@@ -1,6 +1,8 @@
 import sys
 from setuptools import setup, find_packages
 
+version = "0.0.0"
```

Comment:
I was trying to make changes to py-moose-lib and use that in a local moose project. Tried to install the package from a local path, but ran into an error about not finding the version:

<img width="1939" alt="Screenshot 2024-09-16 at 3 15 46 PM" src="https://github.com/user-attachments/assets/cc08b32b-1c22-4791-b255-91f955fc59f4">

Seems like `setup.py` works for builds because the script gives it a version. Not sure if `pip install -e` is how others are doing this locally.